applications/contrastive_phenotyping/contrastive_cli/plot_embeddings.py

```diff
@@ -16,6 +16,13 @@
 )
 dataset
 
+# %%
+# Extract a track from the dataset
+all_tracks_FOV = dataset.sel(fov_name=slice("A/4/0"))
+a_track_in_FOV = all_tracks_FOV.sel(track_id=slice(23))
+print(a_track_in_FOV.sizes())
+# Why is sample dimension ~22000 long after the dataset is sliced by FOV and by track_id?
```

Comment:
@ziw-liu how can I slice the dataset by specified fov_name and specific track_id? The above lines of code don't return the dataset of expected dimensions. I would expect a dataset with ~10-30 samples, but get ~22000 samples.

Reply:
A sample is one pair of feature/projected vector (per node `id` in tracking graph). One `track_id` contains many `id`s.

Reply:
And here by using `slice(23)` you are slicing multiple `track_id`s:

```py
assert len(list(range(26))[slice(23)]) == 23
```

Reply:
Your explanation is not clear to me.
Here is the logic I am following:
* I slice all tracks in FOV "A/4/0" and go from 6500 samples (65000 pairs features/projections) to ~500 samples (track_ids x time points). 
* I slice a track_id in the above dataset to get ~30 samples for specific track_ids.

>  One track_id contains many ids.
In other words, I am trying to index samples by fov_name and track_id

I am not familiar with xarray indexing API. Please implement the above and push your change.

Reply:
This gives 38 samples:
```suggestion
# %%
# Extract a track from the dataset
all_tracks_FOV = dataset.sel(fov_name="/A/4/0")
a_track_in_FOV = all_tracks_FOV.sel(track_id=23)
```

Reply:
>  asert len(list(range(26))[slice(23)]) == 23
I see that I am slicing indices 1:23. and "A/3/0":"A/4/0". Can't figure out how to index using only those coordinate values.

Reply:
thanks @ziw-liu